exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
+
+import sys
+
+FILE_PATH = 1
```

Comment:
this constant variable sounds like a string. why is it equal to 1?
maybe add a comment that describes these constant variables